GetEmailAddress.java

```diff
@@ -0,0 +1,177 @@
+
+import org.jsoup.Connection;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.*;
+import org.jsoup.select.Elements;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class GetEmailAddress {
+
+    private final String PROTOCOL_HTTP = "http://";
+    private final String PROTOCOL_HTTPS = "https://";
+    private String protocol;
+    private String baseUrl;
+    private Set<String> visitedLinks = new HashSet<>();
+    private Set<String> collectedEmails = new HashSet<>();
+
+    public GetEmailAddress(String baseUrl) {
+        this.baseUrl = baseUrl;
+    }
+
+    public void parseEmailAddresses() {
+        setProtocol();
+        getHtmlContent(baseUrl);
+    }
+
+    private void setProtocol() {
+        if (baseUrl.startsWith(PROTOCOL_HTTPS)) {
+            protocol = PROTOCOL_HTTPS;
+        } else {
+            protocol = PROTOCOL_HTTP;
+        }
+    }
+
+    private void getHtmlContent(String webUrl) {
+        String address = addProtocol(webUrl);
+        try {
+
+            Connection.Response response = Jsoup.connect(address).ignoreContentType(true)
+                    .userAgent("Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/535.21 (KHTML, like Gecko) Chrome/19.0.1042.0 Safari/535.21")
+                    .timeout(10 * 1000)
```

Comment:
not familiar with how much time this is

Reply:
It is 10 seconds for each request.

Reply:
Sorry.. I meant to convey a comment would go nice here

Reply:
I agree

Reply:
can you update the PR